README.md

```diff
@@ -2,74 +2,138 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+## Goals
+This is the week 6 weekend challenge for Makers Academy. During this week, we were introduced to Javascript and JQuery.
 
-## The Task
+- Translate last weekend's bowling challenge from Ruby into Javascript.
+- Use JQuery to create a user interface.
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+## My Project
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+## Set Up
+Clone the repository:
+```
+$ git clone https://github.com/emilyalice2708/bowling-challenge-ruby
+```
+Run bundle:
+```
+$ bundle install
+```
 
-A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
+## How To Use
 
-As usual please start by
+Run app.rb:
+```
+$ ruby app.rb
+```
 
-* Forking this repo
+- Visit http://localhost:4567/ in the browser:
+![Homepage](https://i.imgur.com/eBiWLaW.png)
 
-* Finally submit a pull request before Monday week at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday week at 9am.  And since next week is lab week you have a full extra week to work on this.
+- Add to the scorecard by typing a valid number and clicking 'Bowl':
```

Comment:
This readme is excellent, really detailed. The only thing I could think to add is instructions on how to run tests, especially as you have Ruby tests and JS tests